photos.html

```diff
@@ -1,111 +1,203 @@
+                <p class="gallery__likes gallery__likes--icon">96</p>
               </div>
-            </section>
+            </div>
+          </section>
+
+          <section class="gallery__item">
+            <div class="gallery__inner">
+
 
-            <section class="gallery__item">
-              <a class="gallery__image galley__image--bridge"></a>
-              <div class="gallery__inner">
+              <div class="gallery__picture">
+                <picture>
+                  <source media="(max-width: 640px)" srcset="img/photo-bridge-mobile@1x.jpg 1x, img/photo-bridge-mobile@2x.jpg 2x">
+                  <source media="(max-width: 1000px)" srcset="img/photo-bridge-tablet@1x.jpg 1x, img/photo-bridge-tablet@2x.jpg 2x">
+                  <source media="(min-width: 1001px)" srcset="img/photo-bridge-desktop@1x.jpg 1x, img/photo-bridge-desktop@2x.jpg 2x">
+                  <img src="/img/photo-bridge-mobile@1x.jpg" alt="The Devil's Bridge">
+                </picture>
+              </div>
+
+              <div class="gallery__caption">
                 <h3 class="gallery__title">The Devil's Bridge</h3>
                 <p class="gallery__author">Photo by: Anton</p>
-                <p class="gallery__likes">254</p>
+                <p class="gallery__likes gallery__likes--icon">254</p>
               </div>
-            </section>
-          </section>
-
-          <section class="gallery__video">
-            <h2 class="gallery__header-video">Still not convinced?</h2>
-            <p class="gallery__paragraph-video">See the video presentation and more for tickets, until they have risen in price again!</p>
-            <div class="video__inner">
-              <video calss="gallery__video-source" src="" controls preload="auto" poster="">
-                <p class="gallery__vide--error">Your browser doesn't support HTML5 video tag.</p>
-              </video>
             </div>
           </section>
-        </div>
-      </main>
+        </section>
 
-      <footer class="page-footer">
-        <div class="page-footer__inner">
-          <div class="page-footer__tag">
-            <h2 class="page-footer__tag-header">#VISITSEDONA</h2>
+        <section class="video">
+          <div class="video__intro">
+            <h3 class="video__header">Still not convinced?</h3>
+            <p class="video__paragraph">See the video presentation and more for tickets, until they have risen in price again!</p>
           </div>
+          <div class="video__inner">
+            <video calss="video__source" src="" control=false preload="auto" poster="img/video-mobile@1x.jpg">
+              <source>
+              <p class="video__error">Your browser doesn't support HTML5 video tag.</p>
+            </video>
+            <div class="video__control">
+              <div class="video__bar"></div>
+              <div class="video__btn-control">
+                <div class="video__icon video__icon--play" title="Play/Pause video">
```

Comment:
let's make it a button

Reply:
you track line is not responsive and you're missing the circle control on it
![image](https://user-images.githubusercontent.com/1570578/35458313-c227b91a-02d3-11e8-959b-30c47ca4d3a9.png)
![image](https://user-images.githubusercontent.com/1570578/35458381-0245e3b4-02d4-11e8-965c-d46b87830c17.png)

Reply:
I was looking for a dot in the files which you have sent me but I don't have any. Also, I think this should be generated by the js while the video is played.